Triangulation/include/CGAL/Triangulation.h

```diff
@@ -1305,18 +1311,18 @@ operator<<(std::ostream & os, const Triangulation<TT, TDS> & tr)
     if( n == 0 )
         return os;
 
-    size_t i(0);
+    int i(0);
     // write the vertices
```

Comment:
Would be better to leave size_t here and to change to `std::map<Vertex_handle, size_t> index_of_vertex` below.